code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -33,6 +33,7 @@
 	var/material_drop_amount = 2
 	var/delivery_icon = "deliverycloset" //which icon to use when packagewrapped. null to be unwrappable.
 	var/anchorable = TRUE
+	var/lockable = TRUE

```

Comment:
would you mind leaving a comment what this does?

Reply:
do we really need a comment to explain what "lockable" stands for

Reply:
If I saw that var without context of this PR, I would assume it's what allows me to enable and disable the crate from being locked, not just disable an examine message.

Reply:
Why isn't it that for that matter

Reply:
(I think it has to do with the fact that abandoned crates can hold stuff that's too big for a regular crate? idk it's been ages since I looked at what is actually in them)